modules/EvManager/main/car_simulatorImpl.cpp

```diff
@@ -199,8 +199,22 @@ void car_simulatorImpl::register_all_commands() {
         command_registry->register_command("iso_dc_power_on", 0, [this](const CmdArguments& arguments) {
             return this->car_simulation->iso_dc_power_on(arguments);
         });
-        command_registry->register_command("iso_start_v2g_session", 1, [this](const CmdArguments& arguments) {
-            return this->car_simulation->iso_start_v2g_session(arguments, mod->config.three_phases);
+        command_registry->register_command("iso_start_v2g_session", 3, [this](const CmdArguments& arguments) {
```

Comment:
Because of the changes the node red flows in everest-core/config/nodered/ should also be modified. Without this modification the register_command will fail.